packages/bundle-source/README.md

```diff
@@ -60,3 +101,67 @@ the bundle in the event of any stack traces.
 Also, the toplevel `getExport(filePrefix = "/bundled-source")` accepts an
 optional `filePrefix` argument (which is prepended to relative paths for the
 bundled files) in order to help give context to stack traces.
+
+## endoZipBase64 moduleFormat
+
+An Endo (zip, base64) bundle is an object with properties:
+
+- `moduleFormat` is `"endoZipBase64"`
+- `endoZipBase64` is a base 64 encoded zip file.
+- `endoZipBase64Sha512`, if present, is the SHA-512 of the
+  `compartment-map.json` file inside the `endoZipBase64` archive.
+  If the `compartment-map.json` includes the SHA-512 of every module, this is
+  sufficient as a hash of the bundled application for checking its integrity
+  and is consistent regardless of whether the program is extracted from the
+  archive.
+
+To inspect the contents of a bundle in a JSON file:
+
+```
+jq -r .endoZipBase64 | base64 -d | xxd | less
+```
+
+To extract the contents:
+
+```
+jq -r .endoZipBase64 | base64 -d > bundle.json
```

Comment:
calling it `bundle.json` seems odd. I would have thought `bundle.zip`

Reply:
Thanks. Queuing up https://github.com/endojs/endo/pull/1693